deploy/cdk/lib/static-host/pipeline-s3-sync.ts

```diff
@@ -45,6 +45,7 @@ export interface IPipelineS3SyncProps extends PipelineProjectProps {  /**
   readonly graphqlApiUrlKeyPath: string
   readonly graphqlApiKeyKeyPath: string
   readonly maintainMetadataKeyBase: string
+  readonly buildEnvirionment: string
```

Comment:
Sorry I didn't catch this in the other PR. You'll need to change in the other spots too obv
```suggestion
  readonly buildEnvironment: string
```